src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,71 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as currentTodoActions } from '../../features/currentTodo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
 
 export const TodoModal: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```
to fix modal blinking